extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/functionExtractors.scala

```diff
@@ -30,9 +31,12 @@ object FunctionExtractor {
   }
 
   def buildFunctionIdentFromQualifiedName(qualifiedName: String): (String, Option[String]) = {
```

Comment:
How about make it return `Function` class directly?

Reply:
> How about make it return `Function` class directly?

reasonable